draft-mouris-cfrg-mastic.md

```diff
@@ -1130,14 +1130,23 @@ Each output share consists of the truncated payload for each VIDPF prefix,
 flattened into a single vector. Aggregation involves simply adding these up:
 
 ~~~ python
-def aggregate(self,
-              agg_param: MasticAggParam,
-              out_shares: list[list[F]],
-              ) -> list[F]:
-    agg_share = self.empty_agg(agg_param)
-    for out_share in out_shares:
-        agg_share = vec_add(agg_share, out_share)
-    return agg_share
+def agg_init(self, agg_param: MasticAggParam) -> list[F]:
+    return self.empty_agg(agg_param)
```

Comment:
Suggestion: `agg_init()` and `empty_agg()` are functionally equivalent, so we should just get rid of `empty_agg()` and inline it here.

Reply:
fixed in the next PR